rhel/repositoryscanner_test.go

```diff
@@ -55,6 +60,23 @@ func TestRepositoryScanner(t *testing.T) {
 	srv := httptest.NewServer(mux)
 	defer srv.Close()
 
+	esrv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		t.Fatal("external http request invoked when none was expected")
+	}))
+
+	f, err := os.CreateTemp("", "repository-2-cpe.json")
```

Comment:
nit but we have `pkg/tmp.NewFile()` and `.Close()` that will close the file and remove it.

Reply:
another option is using `t.TempDir()` which will auto-close once the test is done

Reply:
A combination of the two seems pretty nice